config/utilities/copyFilesAndFolders.ts

```diff
@@ -0,0 +1,19 @@
+const fse = require('fs-extra')
+/**
+ * copyFilesAndFolders
+ * @description Copies file from copyFilesAndFolders array this is used to copy deprecated and removed values
+ * @param filesAndFolders array of paths
+ * @param source path 
+ * @param destination path 
+ * @returns promise
+ */
+export const copyFilesAndFolders = async (filesAndFolders: string[], source: string, destination: string) => {
+  return Promise.all(filesAndFolders.map(
```

Comment:
Bit hard to read. Can you break this up into smaller units of code and compose them instead?

Reply:
I extracted the `errorHandler` I hope this makes it easier to read.

As we said we once merged we should create an issue to replace this function including the `fse` with our own implementation of using `fs` to avoid the dependency.